Supporting Documents/BS_SD.adoc

```diff
@@ -517,7 +517,8 @@ The following test steps require the developer to provide access to a test platf
 +
 . The evaluator may examine feedback messages shown (e.g. “Finger too wet” in case of fingerprint verification) to understand how the evaluator can present a biometric characteristic that does not meet the <<MBE assessment criteria for samples, assessment criteria for samples>>
 
-. The evaluator may ask the developer how low quality samples can be created if the TOE doesn’t show any feedback messages
+. The evaluator shall review public papers on biometric quality (see <<Related Documents>> for some referenced quality papers) for the appropriate modality to determine how to create low quality samples. The evaluator shall document the papers used for reference in the evaluation report.
+.. The evaluator may ask the developer for assistance if public methods are unsuccessful in creating the low quality samples
```

Comment:
Quality is relative to the algorithm used to measure it. While there's some general criteria for most modalities, they might not reflect a low quality sample for a particular quality algorithm. Below might help?

```suggestion
. The evaluator shall review public papers on biometric quality (see <<Related Documents>> for some referenced quality papers) for the appropriate modality and algorithm to determine how to create low quality samples. The evaluator shall document the papers used for reference in the evaluation report.
.. The evaluator may ask the developer for assistance if public methods are unsuccessful in creating the low quality samples (e.g., the quality method is not public).
```